java/src/main/java/com/powsybl/dataframe/network/adders/Util.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Terminal;
+
+import java.util.function.Consumer;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+public final class Util {
```

Comment:
There is already another Util class. You should give it a more precise name like AdderUtil